editor/components/block-settings-menu/converters.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
```

Comment:
This is a nice example of how much you can achieve using only packages offered by Gutenberg 💯

Reply:
Didn't realize that! :gutenberg-powered: